pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
Oh I see. But why do we want to round the result in DataFrame? Users can round them off after directly on the df using `df.applymap(lambda x: '%.2f' % x))`